### ISSUE TITLE
[Lake][DuckDB] Stabilize pdr lake raw + etl update in duckdb-integration PR

### ISSUE THREAD

**Issue by idiom-bytes** [oceanprotocol/pdr-backend#1036]
### Background / motivation
Right now, in duckdb latest w/ validations, the `lake raw update` command fails on pdr-slots.
![image](https://github.com/oceanprotocol/pdr-backend/assets/69865342/9999d1b0-e652-4c9d-acd5-05941d41b662)

### TODOs / DoD
Fix pdr-slots and make sure that `pdr lake update` command and `pdr etl update` command are completed e2e, and that validations or other problems are not stopping the pipeline.

Tasks:
- [x] Fix validation, schema, or new system problem that's causing GQLDF steps to fail.
- [x] `pdr lake update` command and `pdr etl update` command are completing e2e
- [x] GQLDF pdr-slots is running, **problem is addressed/stability improved**
- [x] pdr-slots and bronze-slots "should just work" (and were working before the validation merge)
- [x] Merge into `duckdb-integration` PR

**Comment by KatunaNorbert** [comment]
`GQLDF pdr-slots + bronze-slots are finally, disabled for now (they were just working, so let's fix them and understand why they broke)`

I don't understand this points, should we disabled this tables?

**Comment by idiom-bytes** [comment]
```I don't understand this points, should we disabled this tables?```

We made this fix this because the validation update broke it... but generally, I want to turn off these extra tables, only have the stuff required for #1000, and then "freeze a release candidate".

**Comment by idiom-bytes** [comment]
I have just tested the `bronze_slots` table and as far as I can tell, it is not working.

I'm also looking at #1000 PR - https://github.com/oceanprotocol/pdr-backend/pull/1007
(1) `bronze_slots` table is disabled, such that it can complete successfully.
(1b) When I look at the PR/changelog, I can see that `bronze_slots` was removed (probably by me) so it could pass
(2) I have tried enabling it,and running it on issue #1000 as well, but I experience the same issues.
(3) I have moved more details and info to ticket https://github.com/oceanprotocol/pdr-backend/issues/975 that is currently documenting various problems with `bronze_slots`

For #1000 - ETL Steps **will not include bronze_slots**
